src/services/following.service.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  getList,
+  addOne,
+  deleteById,
+} from "../repositories/following.repository";
+
+export const getFollowing = async (queryParams: object) => {
+  const result = await getList(queryParams);
+  return result;
+};
```

Comment:
```suggestion
export const getFollowing =  (queryParams: PaginationParameters<never, never>) => {
  return getList(...queryParams.get());
};
```
example in tweet service